module/VuFind/src/VuFind/AjaxHandler/GetRecordTags.php

```diff
@@ -91,7 +93,7 @@ public function handleRequest(Params $params)
             ];
         }
 
-        $viewParams = ['tagList' => $tagList, 'loggedin' => (bool)$this->user];
+        $viewParams = ['tagList' => $tagList, 'loggedin' => (bool)$this->user, 'driver' => "$source|$id"];
```

Comment:
I considered changing the name of `driver`, but the template is used in other places too.